### PR TITLE
Data-Science Resource issue #98

### PR DESCRIPTION
**Pull request by imskr**
issue #98

### CODE REVIEW

File: roles-in-tech/data-science.md [hunk 1]
```diff
@@ -2,90 +2,90 @@
...
 
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
+**LifeCycle of Data Science**
+<p align="center">
+	<img src="/roles-in-tech/lifecycle-datascience.png" alt="data" height="500" width="500">
+</p>
```
Comment (alodahl) [inline roles-in-tech/data-science.md]: Google “markdown cheat sheet” to see how you can insert images using md notation!

**Comment by imskr** [comment]
Thanks for review. I will make changes accordingly.

File: roles-in-tech/data-science.md [hunk 2]
```diff
@@ -2,90 +2,90 @@
...
+- Guided Practice: Unlimited
+- Independent Practice: Unlimited
+- Check for Understanding: 30-45 min
 
 ### Prerequisites
 
 Here are links to lessons that should be completed before this lesson:
 
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
+- [Git version control](https://github.com/Techtonica/curriculum/blob/master/git-version-control/git-version-control.md)
```
[the remark this answers is not in the document]
Comment (imskr) [reply roles-in-tech/data-science.md]: Not necessary though, ok i will remove it

File: roles-in-tech/data-science.md (hunk 1 above)
Comment (imskr) [reply roles-in-tech/data-science.md]: i am using it to align image to center.

**Comment by imskr** [comment]
@alodahl

**Comment by alodahl** [comment]
fixes #98

**Pull request merged**